v03_pipeline/lib/tasks/reference_data/updated_reference_dataset_collection_test.py

```diff
@@ -221,7 +221,7 @@ def test_update_task_with_empty_reference_data_table(
                     paths=hl.Struct(
                         primate_ai='gs://seqr-reference-data/GRCh38/primate_ai/PrimateAI_scores_v0.2.liftover_grch38.ht',
                         cadd='gs://seqr-reference-data/GRCh38/CADD/CADD_snvs_and_indels.v1.6.ht',
-                        clinvar='ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/clinvar.vcf.gz',
+                        clinvar='{protocol}://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/clinvar.vcf.gz',
```

Comment:
@bpblanken By changing the clinvar source path to include a template, the clinvar path in the rdc ht globals changes too. I don't think this will have negative consequences, it could just be confusing. There's other ways to handle replacing 'ftp' with 'https' to get the etag, lmk if you think I should do something else

Reply:
one silly option is that we just use `https` in the download.  it's... slightly slower I think?  but maybe easier to reason about.